docs/topics/projections.rst

```diff
@@ -403,19 +403,23 @@ notification log even when the notification doesn't imply a real entry in the in
         class Created(Event, AggregateRoot.Created):
             pass
 
+        class Updated(Event):
+            def mutate(self, aggregate):
+                aggregate.index_value = self.index_value
+
+        def update_index(self, index_value):
+            self.__trigger_event__(IndexItem.Updated, index_value=index_value)
+
 
     def uuid_from_url(url):
         return uuid.uuid5(uuid.NAMESPACE_URL, url.encode('utf8') if bytes == str else url)
 
 
     # Define indexer.
     class Indexer(object):
```

Comment:
So as i understand correctly Indexer is just a `Projector` that creates event sourced projection `IndexItem`